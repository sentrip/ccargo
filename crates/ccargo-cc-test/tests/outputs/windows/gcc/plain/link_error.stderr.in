C:\Users\Djordje\AppData\Local\Temp\ccqXxn1o.o:main.c:(.text+0x0): multiple definition of `func'
C:\Users\Djordje\AppData\Local\Temp\ccOdnuho.o:warn.c:(.text+0x0): first defined here
collect2.exe: error: ld returned 1 exit status
