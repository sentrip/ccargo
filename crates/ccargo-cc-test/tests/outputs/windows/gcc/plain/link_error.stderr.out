error .text+0x0:  multiple definition of `func'
   --> C:\Users\Djordje\AppData\Local\Temp\ccqXxn1o.o:main.c
error .text+0x0:  first defined here
   --> C:\Users\Djordje\AppData\Local\Temp\ccOdnuho.o:warn.c
